portfolio/index.html

```diff
@@ -0,0 +1,214 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&display=swap" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <nav>
+
+        <ul class="nav-links">
+            <li><a href="#home">home</a></li>
+            <li><a href="#about">about me</a></li>
+            <li><a href="#project">project</a></li>
+            <li><a href="#contact">contact</a></li>
+        </ul>
+    </nav>
+    <main>
+        <div class="container" id="home">
+            <span class="text">Hi, I am Syed</span>
+            <span class="text">A Junior</span>
+            <span class="text">Web Developer</span>
+        </div>
+        <div class="about-me" id="about">
+            <div class="profile-pic">
+                <img src="https://scontent.fbcn4-2.fna.fbcdn.net/v/t39.30808-6/274187526_3028452014075976_5906046956010075197_n.jpg?_nc_cat=107&ccb=1-7&_nc_sid=6ee11a&_nc_ohc=cDgDOG5SNksQ7kNvgEmAay3&_nc_ht=scontent.fbcn4-2.fna&_nc_gid=AznkzAQHT3gcPd8SrL5H60N&oh=00_AYAakF4u_K3RLFG0wGn-dzc5iwtsdaV9zRzJhODz3SYgxQ&oe=670C4BB3"
+                    alt="Your Name">
+            </div>
+            <div class="description">
+                <h1>About Me</h1>
+                <p>I am a strong engineering professional with a Bachelor of Science - BS in Information Technology from
+                    the University of
+                    Sargodha and great ambition to set myself new challenges. I arrived in Barcelona in 2023 and here I
+                    got the opportunity
+                    to learn and develop my programming skills as a Full-stack developer at Migracode Barcelona here I
+                    am continuing to
+                    graduate and study as a Full-stack developer. I love the magic of code, I'm constantly learning, and
+                    seeking to learn
+                    new tools every day that help me become a better professional. I have front-end knowledge of
+                    applying agile
+                    methodologies such as Kanban and Scrum and back-end knowledge with OOP always developing with the
+                    principle of clean
+                    code and applications for tests.</p>
+            </div>
+        </div>
+        <section class="skills-section">
+            <h1>Development Skills</h1>
+            <div class="skills-container">
+                <div class="skill-card">
```

Comment:
I see, you fixed the habit of using numbers in classes. All skill cards have the same class. Well done

Reply:
yes I will avoid using numbers in classes your advice in my mind